### ISSUE TITLE
Include ffmpeg

### ISSUE THREAD

**Issue by mifi** [mifi/editly#23]
Might add the [ffmpeg-static](https://github.com/eugeneware/ffmpeg-static) dependency, then no need for system ffmpeg. (with an option to use a custom ffmpeg path)
But it does not work on raspberry pi, need to see if it crashes npm install or what happens

**Comment by chapmanjacobd** [comment]
looks like ffprobe is separate 

https://github.com/eugeneware/ffmpeg-static/issues/19

??

https://github.com/joshwnj/ffprobe-static

**Comment by mifi** [comment]
Yeah, I'm thinking about building a reusable ffmpeg GitHub Action and a repository that can build ffmpeg for many platforms in many different variants and allow users to fork the repo for their own build configuration, but it's kind of a big job.

**Comment by chapmanjacobd** [comment]
looks like someone has done something similar before:

https://github.com/vot/ffbinaries-api
https://github.com/vot/ffbinaries-node

**Comment by mifi** [comment]
I found that one too, but there seems to be some issues with those builds [not really being static](https://github.com/vot/ffbinaries-prebuilt/issues/7) builds. I haven't tried them myself and I'm not sure where he gets the builds from, but I think that the official builds are preferred because they are well tested by a lot of people.
And it seems kind of abandoned, also it doesn't have the latest 4.2.2 version: https://github.com/vot/ffbinaries-prebuilt/releases

**Comment by chapmanjacobd** [comment]
oh... for some reason I thought this repo https://github.com/vot/ffbinaries-api included the code for automatically compiling from nightly builds or something :/

**Comment by mifi** [comment]
no. it just [contains references to downloads from the other repo](https://github.com/vot/ffbinaries-api/blob/master/json/4.2.json). So the question is, how are the ffmpeg-prebuilt being built. I've [created an issue for that](https://github.com/vot/ffbinaries-prebuilt/issues/9)

**Comment by maksym-plotnikov** [comment]
@mifi since You\re discussing FFMPEG builds we\re using these:
https://johnvansickle.com/ffmpeg/
And they're always brings an error to console because `editly` complains 
'Unknown version string' since they don't have same build sting as official FFMPEG builds. Can You please take a look on these?

**Comment by mifi** [comment]
Ohh. Are you using "release: 4.3.1"? or "git master" ?

**Comment by mifi** [comment]
Please try 4.3.1 and see if you get the same error

**Comment by cyrilgupta** [comment]
FFMPeg binaries on windows don't have the same version string. I am getting this - N-101088-g6f80953554

**Comment by mifi** [comment]
where did you download ffmpeg from?
